docs/usage.rst

```diff
@@ -302,6 +303,99 @@ Alice is the former owner:
         transfer_asset_id = transfer_tx['asset']['id']
 
 
+
+Recap: Asset Creation & Transfer
+--------------------------------
+
+.. code-block:: python
+
+	from bigchaindb_driver import BigchainDB
+	from bigchaindb_driver.crypto import generate_keypair
+	from time import sleep
+	from sys import exit
+	
+	alice, bob = generate_keypair(), generate_keypair()
+	
+	bdb = BigchainDB('http://localhost:59984/api/v1')
+	
+	bicycle_asset = {
+		'data': {
+			'bicycle': {
+				'serial_number': 'abcd1234',
+				'manufacturer': 'bkfab'
+			},
+		},
+	}
+	
+	bicycle_asset_metadata = {
+		'planet': 'earth'
+	}
+	
+	prepared_creation_tx = bdb.transactions.prepare(
+		operation='CREATE',
+		signers=alice.public_key,
+		asset=bicycle_asset,
+		metadata=bicycle_asset_metadata
+	)
+	
+	fulfilled_creation_tx = bdb.transactions.fulfill(
+		prepared_creation_tx,
+		private_keys=alice.private_key
+	)
+	
+	sent_creation_tx = bdb.transactions.send(fulfilled_creation_tx)
+	
+	txid = fulfilled_creation_tx['id']
+	
+	trials = 0
+	while bdb.transactions.status(txid).get('status') != 'valid' and trials < 60:
```

Comment:
This line will probably fail at first. The latest version of this page (`usage.rst`) puts a try/except inside the loop. (It also uses 100 instead of 60, but that doesn't matter.) Are you working from the latest `master` branch?